_pages/robot/2018.html

```diff
@@ -1,44 +1,48 @@
-        <li><img src="/assets/img/first.svg">2017<br> North VA Innovation in Control</li>
-        <li><img src="/assets/img/first.svg">2017<br> North MD Creativity Award</li>
-        <li><img src="/assets/img/first.svg">2017<br> Chesapeake Innovation in Control</li>
-    </ul>
-    <img src="/assets/img/robots/2017.jpg" alt="The 2017 robot being carried onto the ground.">
-    <p>The 2017 game is proving to be one of the most challenging to date for our experienced team. Designed by Disney, FIRST STEAMWORKS aims to challenge teams to create mechanisms capable of delivering 11 inch diameter gears to human players stationed on the Jules Verne inspired airship. Teams also have the task of shooting 5 inch diameter wiffle balls to add fuel to the boiler, sending steam to fuel the airship. For a more detailed description of the game, see the <a href="https://www.youtube.com/watch?v=EMiNmJW7enI">the official video reveal</a>.</p>
+    <h1>2018 Robot</h1>
+    <iframe src="https://www.youtube.com/v/jqxHeneuZwA"></iframe>
+    <!--TODO: add images-->
+    <p>The 2018 game's fusion of 20th century arcade gamming and 21st century robotics is possibly one of the most challenging scenario to date. For a more detailed description of the game, see the <a href="https://youtu.be/HZbdwYiCY74">the official video reveal</a>.</p>
     <h5>Robot Abilities</h5>
     <ul>
-        <li><strong>Swerve Drive:</strong> Our new swerve system allowed each wheel of our robot to run independently in full 360 degree motion. This gives the robot unprecedented speed and agility allowing us to deliver gears in under 15 seconds and offering a critical ingame advantage.</li>
-        <li><strong>High Goal Shooter:</strong> Our robot had the capacity to collect fuel from the delivery station in a top-mounted hopper leading to our side-positioned flywheel. This allows the robot to shoot 10 balls in autonomous if necessary and contribute to the alliance with the goal of flying the airship. The robot could hold 18 balls and are able to achieve 90% accuracy in testing. After our first competition, we removed our fuel shooter to make room for extra climbing CIMs.</li>
-        <li><strong>Pneumatic Gear Picker:</strong> Pneumatic Gear Picker: Unlike most robots, we have the ability to pick up gears off the floor in an active delivery system. Using a pneumatically actuated hook we are able to quickly collect gears deposited from the delivery station and place them on the airship’s springs. This allows up to be very versatile and collect any gears left by other teams, maximizing efficiency of the alliance. </li>
-        <li><strong>Climber: </strong>Utilizing a massive gear reduction and milled slots into a rotating plate, we are able to latch onto a knot, pulling our 110 pound robot up into the scoring an onto the rising airship.</li>
+        <li><strong>Arcade Drive:</strong> Fittingly, arcade drive will be our drive method of choice this year. This allows</li>
```

Comment:
Arcade drive is our user input mode, while the chassis is a tank drive configuration

Reply:
Just making sure you’re saying what you want to

Reply:
Yup. Don't worry. I'll clarify that though.